src/components/structures/RoomStatusBar.tsx

```diff
@@ -192,13 +192,19 @@ export default class RoomStatusBar extends React.PureComponent<IProps, IState> {
 
         let consentError = null;
         let resourceLimitError = null;
+        let accountSuspendedError = null;
         for (const m of unsentMessages) {
-            if (m.error && m.error.errcode === 'M_CONSENT_NOT_GIVEN') {
-                consentError = m.error;
-                break;
-            } else if (m.error && m.error.errcode === 'M_RESOURCE_LIMIT_EXCEEDED') {
-                resourceLimitError = m.error;
-                break;
+            if (m.error) {
```

Comment:
instead of nesting, let's use the `?` operator:

`if (m.error?.errcode === 'whatever')`